vignettes/advantages.Rmd

```diff
@@ -7,63 +7,70 @@ vignette: >
 
 ## Easier deployment
 
 CI services can be used to automatically build and deploy files.
 This means that these services can push certain files created during the build to repositories (GitHub, GitLab, etc.).
-Use cases can be
+Possible use cases are:
 
-- changed site contents of _pkgdown_ site
+- changed site contents of a _pkgdown_ site
 - updated .Rd files of a package (by calling `devtools::document()` before)
 - an automatic generated [test summary page](https://github.com/yonicd/covrpage) of a package 
 
-When the goal is to use CI services for deployment, some preparatory work needs to be done.
-*tic* takes care of all of this (setting up SSH keys, granting permission for the repo on GitHub and the CI services, creation of a personal Personal Access Token (PAT)).
-When using `travis::use_ci()` the whole preparation process is streamlined and only needs to be called once for every repo.
+If _tic_ should be used for deployment, some preparatory work is required.
+Luckily, _tic_ takes care of all of this (setting up SSH keys, granting permission for the repo on GitHub and the CI services, creation of a personal Personal Access Token (PAT)).
+When calling `use_ci()`, the streamlined preparation process is run.
+This is only needed once per repository.
 
-For more information on automatic deployment in *tic*, take a look [here](deployment.html).
+For more information on automatic deployment in _tic_, take a look [here](deployment.html).
 
 ## Support for various R projects
 
 *tic* was developed with the aim to not only simplify R package development using CI services but also to support other R project types (bookdown, blogdown, drat, website and figshare deployment). 
 
 Each of these projects has its own standardized structure. 
-*tic* detects this structure and adds templates tailored towards this specific project type to the repository when calling `usethis::use_ci()`.
+*tic* detects this structure and adds templates tailored towards this specific project type to the repository when calling `use_tic()`.
 See [here](tic.html#example-projects) for a list of supported project types including links to minimal example repositories. 
 
 ## CI-Agnostic workflows
 
 What does "CI-Agnostic" mean and why do we need it?
 
-You may have heard that multiple CI services exist in the wild.
-Some only support specific operating systems or certain languages and for some you need to pay a monthly fee to use them.
-Each system has its own way to specify the setup. 
-Often it is a YAML file stored at the top-level directory of your repo.
+You may know that Travis CI is not the only company offering (free) CI solutions.
+Theoretically, R could be run on any CI system.
+Due to historic reasons, some folks from the R community started on Travis implementing an easy way for R package checking.
```

Comment:
How does this paragraph contribute to the story told in this section? It's already mentioned in the "Less dependency..." section below.